firmware/library/L3_HAL/Piezo_Driver.hpp

```diff
@@ -0,0 +1,39 @@
+// This driver is a layout for the piezo buzzer controller implementation
+// It contains the interface, which is inherited by the "Buzzer" class
+// which implements the functions in the interface
+// The constructor is responsible for initializing the port
+
+#pragma once
+
+// import only the necessary libraries to make this interface function
+#include <cstdint>
+
+// import the lpc_pwm class from SJOne library as our PWM driver
```

Comment:
Since it is clear that you are importing this file, no need to comment that you are including this file